assists/cdo.py

```diff
@@ -51,144 +51,6 @@ def is_compat(node, compat_id):
     return ""
 
 
-def do_expand_cdo_flags(tgt_node):
```

Comment:
Can we tweak the commit message to indicate why this is being done ? Is it in preparation for future changes ? cleanup ? import simplification ? something else ?

Reply:
The reasoning for this is mainly code cleanup and better code organization.

do_expand_cdo_flags() and other helpers are  mainly doing the YAML to DTS flags expansion for domains/subsystems. This code correctly belongs in assists/subsystem.py (xlate-yaml) plugin given its role in the subsystem expansion.
That's why this change.

I will update the commit message to add these details.

Reply:
perfect. Thanks!